ovos_translate_server/__init__.py

```
@@ -65,9 +82,29 @@ def __init__(self, tx_plugin: str, detect_plugin: Optional[str] = None) -> None:
+        plugin) reject an empty source outright, which made this route fail for
+        them. When a detection plugin is configured, use it to resolve the
+        source first, then hand both languages to the translator. Falls back to
+        letting the engine try on its own when no detector is available.
+        """
+        if self.detect is not None:
+            try:
+                src_lang = self.detect.detect(utterance)
+                if src_lang:
+                    return self.tx.translate(utterance, target=tgt_lang,
+                                             source=src_lang)
+            except Exception as err:  # detection is best-effort
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)